### ISSUE TITLE
delayed start

### ISSUE THREAD

**Issue by ksaitor** [glennreyes/react-countup#87]
Would be great to have a `delay` attribute in seconds (or milliseconds) that would postpone the start of the animation.
## Example:
```jsx
<CountUp start={0} end={100} delay={3} />
```
— would would delay the animation for 3 seconds

**Comment by glennreyes** [comment]
Totally! I’d prefer to keep the CountUp component as minimal as possible. What do you think about that as a wrapper component? Eg. `<Delay />`

**Comment by ksaitor** [comment]
🤔 Imho code will look clumsy and might be misread. E.g.
```jsx
<Delay value={3}>
  <CountUp start={0} end={100} />
</Delay>
```
☝️ 2 more lines and 16 more characters compared to `delay={3}` attribute.
Also, if I were a 3rd person, reviewing the code, I'd think that whole `<CountUp />` component rendering will be delayed by 3 seconds and nothing will be displayed for 3 seconds…

However the intention of the `delay` is to show `start` value for 3 seconds, before starting animate it.

**Comment by glennreyes** [comment]
Yeah good points! Let's add the `delay` prop. Would you mind adding a PR?
